logprep/processor/selective_extractor/rule.py

```diff
@@ -60,18 +85,7 @@ def extracted_field_list(self) -> List[str]:
         --------
         extracted_field_list: a list with extraction field names
         """
-        return self._extract.get("extracted_field_list")
-
-    def _add_from_file(self):
-        if not os.path.isfile(self._extract_from_file):
-            raise SelectiveExtractorRuleError("extract_from_file is not a valid file handle")
-        extract_list = self._extract.get("extracted_field_list")
-        if extract_list is None:
-            extract_list = []
-        with open(self._extract_from_file, "r", encoding="utf8") as extract_file:
-            lines_from_file = extract_file.read().splitlines()
-            extract_list = list(set([*extract_list, *lines_from_file]))
-        self._extract["extracted_field_list"] = extract_list
+        return self._config.extract.get("extracted_field_list")
 
     def __eq__(self, other: "SelectiveExtractorRule") -> bool:
```

Comment:
Do we still need this method? or is it necessary because of the `set`-cast?

Reply:
is is needed, because it is using the full config instead. This would lead to other results